src/utils/config_filters.cpp

```diff
@@ -55,6 +55,9 @@ bool utils::config_filters::int_matches_if_present(const config& filter, const c
 	if(filter[attribute].empty()) {
 		return true;
 	}
+	if(cfg[attribute].empty()) {
+		return false;
+	}
```

Comment:
`unsigned_matches_if_present` needs to be either updated as well, or just removed since nothing uses it anyway.

Reply:
> `unsigned_matches_if_present` needs to be either updated as well, or just removed since nothing uses it anyway.

function updated, but i prefer what it remain in case where someone would using this in future.